packages/css/src/dialog.css

```diff
@@ -17,17 +17,26 @@
   /* Internal */
   --_dsc-dialog-icon-top-right-margin: calc(var(--dsc-dialog-spacing) * -1 + var(--dsc-dialog-icon-spacing));
 
+  /* Center with top+left+translate to support placement inside display: flex */
+  animation: _fadein .2s forwards;
```

Comment:
```suggestion
  animation: fade-in .2s forwards;
```
![image](https://github.com/user-attachments/assets/e6f06fa3-4867-4567-95fe-1bd18e4c3a07)

Reply:
oh takk! well spotted! her kom det med litt av det som fixet hos oss - skal rette <3